src/ui/FormWidgets/TextInput.ts

```diff
@@ -33,13 +35,20 @@ export interface ITextInputOptions {
    * A custom aria-label attribute to add to the `TextInput`'s HTML element.
    */
   ariaLabel?: string;
+  /**
+   * The name or svg source of an icon to display inside the input at its beginning.
```

Comment:
Do we need to support svg urls for this PR? Or is a name enough for now? I prefer to add things only once we need them.

Reply:
We don't need it, I'll remove it.